packages/core-data/src/entity-types/post-type.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * Internal dependencies
+ */
+import type { Context, OmitNevers } from './helpers';
+import type { BaseEntityRecords as _BaseEntityRecords } from './base-entity-records';
+
+declare module './base-entity-records' {
+	export namespace BaseEntityRecords {
+		/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
+		export interface PostType< C extends Context > {
```

Comment:
By the way, post types are covered in `type.ts` already I think

Reply:
Didn't know that. Thanks. Updated.